File: python-client-sdk/src/orbapi/models/shared/security.py
import dataclasses



@dataclasses.dataclass
class SchemeVesselAPIToken:
    api_key: str = dataclasses.field(metadata={'security': { 'field_name': 'vessel-api-token' }})
    

@dataclasses.dataclass
class Security:
    vessel_api_token: SchemeVesselAPIToken = dataclasses.field(metadata={'security': { 'scheme': True, 'type': 'apiKey', 'sub_type': 'header' }})
    

File: typescript-client-sdk/src/sdk/models/shared/security.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";



export class SchemeVesselApiToken extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "security, name=vessel-api-token" })
  apiKey: string;
}


export class Security extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "security, scheme=true;type=apiKey;subtype=header" })
  vesselAPIToken: SchemeVesselApiToken;
}
